### ISSUE TITLE
Yarn fails to build npm package prepush-hook on windows

### ISSUE THREAD

**Issue by H34D** [yarnpkg/yarn#1239]
Using Yarn 1.16.0

**Do you want to request a _feature_ or report a _bug_?**
Bug

**What is the current behavior?**
Fails on building the package, npm does not fail

**If the current behavior is a bug, please provide the steps to reproduce.**
yarn install prepush-hook or yarn install with prepush-hook in package.json

[4/4] Building fresh packages...
[1/7] ⠂ fsevents
[2/7] ⠂ buffertools
[3/7] ⠂ node-sass
[4/7] ⠂ prepush-hook
error ......\node_modules\prepush-hook: Command failed.
Exit code: 1
Command: C:\WINDOWS\system32\cmd.exe
Arguments: /d /s /c hook_install
Directory: .....\node_modules\prepush-hook
Output:
The command "hook_install" ist entweder falsch geschrieben oder konnte nicht gefunden werden.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

it looks like yarn is unable to find the binary file. Inside the package.json of prepush-hook are the following configurations:

"bin": {
    "hook_install": "./bin/install",
    "hook_uninstall": "./bin/uninstall",
    "pre-push": "./bin/pre-push"
},
"scripts": {
    "install": "hook_install",
    "uninstall": "hook_uninstall"
},

i think this is sort of uncommon.

**What is the expected behavior?**
Yarn should build the package, like npm does

**Please mention your node.js, yarn and operating system version.**
node 6, yarn 1.16, windows 10 x64

**Comment by rgoldfinger** [comment]
Seeing the same issue with the `git-hooks` package:
```
> yarn add git-hooks                                                                                                                                                 rg-leave ✭ ✱ ◼
yarn add v0.16.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
[1/1] ⠄ git-hooks
[-/1] ⠄ waiting...
[-/1] ⠄ waiting...
[-/1] ⠄ waiting...
error /Users/rgoldfinger/remind/r101-react-native/node_modules/git-hooks: Command failed.
Exit code: 127
Command: sh
Arguments: -c git-hooks --install
Directory: /Users/rgoldfinger/remind/r101-react-native/node_modules/git-hooks
Output:
sh: git-hooks: command not found
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

I wonder if https://github.com/yarnpkg/yarn/pull/1210 will fix it?

**Comment by kittens** [comment]
Fixed via #1210.

**Comment by H34D** [comment]
just tried v0.17.6 and the Problem still exists
